app/controllers/api/reservations_controller.rb

```diff
@@ -4,7 +4,14 @@ def index
     render json: @reservations
   end
 
-  def create; end
+  def create
+    @reservation = Reservation.new(reservation_params)
+    if @reservation.save
```

Comment:
def reserveation_params on line 18 is a typo error please fix to ensure that the code is consistent